File: NastyHacks.py
times = int(input())
for i in range (times):
    r, e, c = map(int, (input().split()))
    if(r > e-c):
        print("do not advertise")
    elif(r < e-c):
        print("advertise")
    else:
        print("does not matter")

File: nastyhacks.cpp
#include <iostream>
using namespace std;

int main()
{
    int times;
    cin>> times;
    for(int i =0; i < times; i++)
    {
        int r, e, c;
        cin>> r>> e>> c;
        if(r > e-c)
            cout<< "do not advertise"<< endl;
        else if(r < e-c)
            cout<< "advertise"<< endl;
        else    
            cout<< "does not matter"<< endl;
    }
    return 0;
}